### PR TITLE
feature/fix-bug-filter-user 

### PR DESCRIPTION
**Pull request by AlexeyGrishechkin**
fix search handler and filter users

### CODE REVIEW

File: src/pages/HomepageManager/index.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ const HomepageManager = ({
       }
```
Comment (alexanderchernetsky) [inline src/pages/HomepageManager/index.tsx:112]: Is it a new feature??? Or it's a bug fix? Please rename the branch

File: src/pages/HomepageManager/index.tsx [hunk 2]
```diff
@@ -112,7 +112,7 @@ const HomepageManager = ({
...
     } else {
       history.push(
-        `${createSearchString({ ...getUrlParams(), [`${selectName}`]: value })}`
+        `${createSearchString({  [`${selectName}`]: value })}`
```
Comment (alexanderchernetsky) [inline src/pages/HomepageManager/index.tsx:115]: You're resetting another filter. This is incorrect behaviour! https://streamable.com/sok5jo

**Review by alexanderchernetsky — CHANGES_REQUESTED** [review]

File: src/pages/HomepageManager/index.tsx (hunk 1 above)
Comment (AlexeyGrishechkin) [reply src/pages/HomepageManager/index.tsx:112]: fixed

File: src/pages/HomepageManager/index.tsx (hunk 2 above)
Comment (AlexeyGrishechkin) [reply src/pages/HomepageManager/index.tsx:115]: fixed

File: src/pages/HomepageManager/index.tsx (hunk 1 above)
Comment (alexanderchernetsky) [reply src/pages/HomepageManager/index.tsx:112]: not fixed. the name of the branch is still incorrect. It should be sth like: bugfix/user-filter

**Pull request closed without merging**